test/test_mps.py

```diff
@@ -7218,6 +7218,13 @@ def test_bernoulli(self):
         mps_out = torch.bernoulli(all_ones)
         self.assertEqual(mps_out, all_ones)
 
+        # Check it works for different dtypes
+        for dtype in [torch.float16, torch.int8, torch.int32, torch.int16, torch.int64]:
```

Comment:
Change looks good. 
Can we please add test for random op as well. It has fixes for Bernoulli and random